dd-trace-api/src/test/groovy/datadog/trace/api/ConfigTest.groovy

```diff
@@ -292,6 +302,7 @@ class ConfigTest extends Specification {
     environmentVariables.set(DD_SERVICE_NAME_ENV, "still something else")
     environmentVariables.set(DD_WRITER_TYPE_ENV, "LoggingWriter")
     environmentVariables.set(DD_TRACE_AGENT_PORT_ENV, "777")
+    environmentVariables.set(SIGNALFX_DB_STATEMENT_MAX_LENGTH, "105")
```

Comment:
I think the point of this suite is to ensure system properties take precedence over env vars if both set.  Mind adding a different property value and assertion like the other config options?

Reply:
Oh, I actually removed that. I am not sure, do we want system properties to over ride the env vars for the db_statement_length? Since the env var (if available) will be specified by the user?